runtimes/rust/lbr-prelude/build.nix

```diff
@@ -0,0 +1,16 @@
+{ inputs, ... }: {
+  imports = [ inputs.nci.flakeModule ];
+  perSystem = { pkgs, system, inputs', config, ... }:
+    let crateName = "lbr-prelude";
+    in {
+      nci.projects.${crateName}.path = ./.;
+      nci.crates.${crateName} = { };
+
+      devShells."dev-${crateName}-rust" = config.nci.outputs.${crateName}.devShell.overrideAttrs (old: {
+        packages = (old.packages or [ ]) ++ [ pkgs.rust-analyzer ];
```

Comment:
Is this the LSP?

Reply:
Yes (this config is now overwritten in #113)